### ISSUE TITLE
Secondary Priorities in Phase 1

### ISSUE THREAD

**Issue by gehalber** [everythingtype/jsa#6]
In order to save time, here is a list of things we can do at a later phase in the process:

Projects
• The Related Projects animation. Initially, this could simply be below the last image in the scroll.
• Slideshow view

News
• The carousel. For now, this image placement could simply be a single image.

Publications
• We could wait on the Featured Essays & Featured Articles
• For books, like News, we could wait on the carousel

Mobile
• This can be a part of Phase 2 as well
